docs/patronictl.rst

```diff
@@ -0,0 +1,1963 @@
+
+    Useful for scripts.
+
+.. _patronictl_edit_config_examples:
+
+Examples
+""""""""
+
+Change ``max_connections`` Postgres GUC:
+
+.. code:: text
+
+    patronictl -c postgres0.yml edit-config batman --pg max_connections="150" --force
+    ---
+    +++
+    @@ -1,6 +1,8 @@
+    loop_wait: 10
+    maximum_lag_on_failover: 1048576
+    postgresql:
+    +  parameters:
+    +    max_connections: 150
+      pg_hba:
+      - host replication replicator 127.0.0.1/32 md5
+      - host all all 0.0.0.0/0 md5
+
+    Configuration changed
+
+Change ``loop_wait`` and ``ttl`` settings:
+
+.. code:: text
+
+    patronictl -c postgres0.yml edit-config batman --set loop_wait="15" --set ttl="45" --force
+    ---
+    +++
+    @@ -1,4 +1,4 @@
+    -loop_wait: 10
+    +loop_wait: 15
+    maximum_lag_on_failover: 1048576
+    postgresql:
+      pg_hba:
+    @@ -6,4 +6,4 @@
+      - host all all 0.0.0.0/0 md5
+      use_pg_rewind: true
+    retry_timeout: 10
+    -ttl: 30
+    +ttl: 45
+
+    Configuration changed
+
+Remove ``maximum_lag_on_failover`` setting from dynamic configuration:
+
+.. code:: text
+
+    patronictl -c postgres0.yml edit-config batman --set maximum_lag_on_failover="null" --force
+    ---
+    +++
+    @@ -1,5 +1,4 @@
+    loop_wait: 10
+    -maximum_lag_on_failover: 1048576
+    postgresql:
+      pg_hba:
+      - host replication replicator 127.0.0.1/32 md5
+
+    Configuration changed
+
+.. _patronictl_failover:
+
+patronictl failover
+^^^^^^^^^^^^^^^^^^^
+
+.. _patronictl_failover_synopsis:
+
+Synopsis
+""""""""
+
+.. code:: text
+
+    failover
+      [ CLUSTER_NAME ]
+      [ --group CITUS_GROUP ]
+      [ { --leader | --primary | --master } LEADER_NAME ]
+      --candidate CANDIDATE_NAME
+      [ --force ]
+
+.. _patronictl_failover_description:
+
+Description
+"""""""""""
+
+``patronictl failover`` performs a manual failover in the cluster.
+
+It is designed to be used when the cluster is not healthy, e.g.:
+
+- There is no leader; or
+- There is no synchronous standby available in a synchronous cluster.
+
+.. note::
+    Nothing prevents you from running ``patronictl failover`` in a healthy cluster. However, we recommend using ``patronictl switchover`` in those cases.
+
+.. warning::
+    Triggering a failover can cause data loss depending on how up-to-date the promoted replica is in comparison to the primary.
+
+.. _patronictl_failover_parameters:
+
+Parameters
+""""""""""
+
+``CLUSTER_NAME``
+    Name of the Patroni cluster.
+
+    If not given, ``patronictl`` will attempt to fetch that from the ``scope`` configuration, if it exists.
+
+``--group``
+    Perform a failover in the given Citus group.
+
+    ``CITUS_GROUP`` is the ID of the Citus group.
+
+``--leader`` / ``--primary`` / ``--master``
+    Indicate who is the expected leader at failover time.
+
+    If given, a switchover is performed instead of a failover.
+
+    ``LEADER_NAME`` should match the name of the current leader in the cluster.
```

Comment:
It is getting deprecated, should we still document it?

Reply:
I guess so. I thought of adding a deprecation notice. We can remove it later, once we remove this altogether.